src/test/dir.proj

```diff
@@ -102,10 +106,9 @@
       <IsCrossArch Condition="'$(TargetArchitecture)' == 'arm' or '$(TargetArchitecture)' == 'arm64' or '$(TargetArchitecture)' == 'armel'">true</IsCrossArch>
       <TestResultsXml>$(TestsOutputDir)$(TestProjectFilename)-testResults.xml</TestResultsXml>
     </PropertyGroup>
-
-    <Exec Command="$(DotnetTestToolCommand) test $(TestArgs) -xml $(TestResultsXml) -notrait category=failing"
-          WorkingDirectory="$(TestProject)"
-          EnvironmentVariables="NUGET_PACKAGES=$(PackagesDir);TEST_ARTIFACTS=$(SystemPathTestsOutputDir);TEST_TARGETRID=$(TestTargetRid);BUILDRID=$(TargetRid)"
+    <Exec Command="$(DotnetTestToolCommand) test $(TestArgs) --logger &quot;trx;LogFileName=$(TestResultsXml)&quot;"
+          WorkingDirectory="$(TestWorkingDirectory)"
+          EnvironmentVariables="NUGET_PACKAGES=$(PackagesDir);TEST_ARTIFACTS=$(SystemPathTestsOutputDir);TEST_TARGETRID=$(TestTargetRid);BUILDRID=$(TargetRid);MNA_VERSION=$(ProductVersion)"
```

Comment:
A couple things:

* The test runner doesn't allow you to pass variables through to Xunit.
* These values are actually used within the test code, and I'm not aware of a way to get Xunit to pass that sort of thing through, anyways.

I agree that these environment variables are awful, though 😞